frontend/src/Frontend/UI/Dialogs/AccountDetails.hs

```diff
@@ -117,8 +120,10 @@ uiAccountDetailsOnChainImpl netname (name, chain, details, account) onClose = Wo
               for_ ksKeys $ \key -> uiInputElement $ def
                 & initialAttributes %~ Map.insert "disabled" "disabled" . addToClassAttr "labeled-input__input labeled-input__multiple"
                 & inputElementConfig_initialValue .~ keyToText key
-          AccountGuard_Other g ->
-            void $ displayText (pactGuardTypeText $ Pact.guardTypeOf g) (renderCompactText g) ""
+          AccountGuard_Other g -> case g of
+            (Pact.GKeySetRef (Pact.KeySetName name)) -> do
```

Comment:
I wonder if we should abstract over keyset refs at this level and do the work of querying the underlying keyset so we can be more informative to the user.  What do you think @sirlensalot?